setup.py

```diff
@@ -4,16 +4,24 @@
 # Parse the version from the mapbox module.
 with open('mapboxcli/__init__.py') as f:
     for line in f:
-        if line.find("__version__") >= 0:
-            version = line.split("=")[1].strip()
-            version = version.strip('"')
-            version = version.strip("'")
+        if "__version__" in line:
+            version = line.split("=")[1].strip().strip('"').strip("'")
             continue
 
 setup(name='mapboxcli',
       version=version,
       description="Command line interface to Mapbox Web Services",
-      classifiers=[],
+      classifiers=['Development Status :: 5 - Production/Stable',
+                   'Environment :: Console',
+                   'Intended Audience :: Developers',
+                   'License :: OSI Approved :: MIT License',
+                   'Programming Language :: Python',
+                   'Programming Language :: Python :: 2.7',
```

Comment:
👍  Thanks. Let's also add 2.6 and `Programming Language :: Python :: Implementation :: PyPy`